### ISSUE TITLE
[Bug]: zli image output column headers are not aligned with output and unfriendly to cli parsing

### ISSUE THREAD

**Issue by raharper** [project-zot/zot#1591]
### zot version

{"level":"info","distribution-spec":"1.1.0-dev","commit":"v2.0.0-rc5-0-g8cc5137","binary-type":"-lint-metrics-mgmt-scrub-search-sync-ui-userprefs","go version":"go1.20.5","time":"2023-07-07T09:57:33-05:00","message":"version"}

### Describe the bug

`zli image repoName --name foo` produces hard-to-read output due to the header columns not being aligned with the actual output data.

```
IMAGE NAME  TAG       OS/ARCH	      DIGEST	  SIGNED      SIZE
aaaaaa/bbbbbbb  8-dev17-213-g40ecfd036  linux/amd64	 9dd8c9ed    false	 264MB
aaaaaa/bbbbbbb  8-dev17-201-gda3c8bebc-squashfs  linux/amd64	  e6267429    false	  249MB
```

Using something like `column -t` should just work but the 'IMAGE NAME'  has whitespace and breaks the alignment

```
IMAGE            NAME                             TAG          OS/ARCH   DIGEST  SIGNED  SIZE
aaaaaa/bbbbbbbb  8-dev17-184-g7305cd43d-squashfs  linux/amd64  3f0900f9  false   249MB
aaaaaa/bbbbbbbb  8-dev17-201-gda3c8bebc-squashfs  linux/amd64  e6267429  false   249MB
```

Further, the response from zot is not sorted, so re-running the same query results in different output order.

```


### To reproduce

1. Configuration
2. Client tool used
3. Seen error


### Expected behavior

```
IMAGE            TAG                              OS/ARCH      DIGEST    SIGNED  SIZE
aaaaaa/bbbbbbbb  8-dev17-186-gf8fcb1a8c           linux/amd64  a4a4db0c  false   264MB
aaaaaa/bbbbbbbb  8-dev17-188-gc957bbb39           linux/amd64  7375c7ce  false   264MB
aaaaaa/bbbbbbbb  8-dev17-188-gc957bbb39-squashfs  linux/amd64  f897f638  false   249MB

```

### Screenshots

_No response_

### Additional context

_No response_

**Comment by rchincha** [comment]
Rename "IMAGE NAME" to "REPOSITORY" similar to docker output.

**Comment by andaaron** [comment]
@raharper some notes:
    - I will rename "IMAGE NAME" to "REPOSITORY" in order to make the header easier to parse
    - The order of the images cannot be predicted if zot is getting them 1 by 1 using the REST API for manifests, so they cannot be sorted when printed. We could wait on all calls to return but that may take minutes, and printing partial results as they become available is better.
    - The order of the images can be predicted when relying on the zot specific search API, but that is not available in all zot servers depending on build options. We are planning to implement sorting in a separate PR - see the work under https://github.com/project-zot/zot/pull/1577
    - With regards to the column widths/alignments that was discussed before, and the issue is we don't know the values beforehand for the REST API based responses. As mentioned above printing partial results as they become available is better.
    - The column widths/alignments are already working properly for responses based on the search API - see https://github.com/project-zot/zot/pull/851

**Comment by andaaron** [comment]
Looks like we do have a server-side bug with the order of tags not being consistent when using the search API. Will patch the specific call used by the images command, but we'll need a more flexible solution for https://github.com/project-zot/zot/pull/1577
